tests/policy.bats

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bats
+
+setup() {
+    load ../helpers/helpers.sh
+    wait_pods
+}
+
+teardown_file() {
+    load ../helpers/helpers.sh
+    kubectl delete pods --all
+    kubectl delete admissionpolicies,clusteradmissionpolicies --all -A
+    kubectl delete ns shouldbeignored
+}
+
+# Number of policies included in the recommended policies
+POLICY_NUMBER=6
+
+@test "[Policy tests] Install default policies in protect mode" {
+    helmer set kubewarden-defaults \
+        --set recommendedPolicies.enabled=True \
+        --set recommendedPolicies.defaultPolicyMode=protect \
+        --set recommendedPolicies.skipAdditionalNamespaces[0]='shouldbeignored'
+
+    # Wait for policies be enforced
+    wait_policies PolicyUniquelyReachable
+
+    # Check we get the correct recommended policies number
+    kubectl --no-headers=true get ap,cap,apg,capg -A | wc -l | grep -q $POLICY_NUMBER 
+}
+
+# Create pod-privileged policy to block CREATE & UPDATE of privileged pods
+@test "[Policy tests] Check that pod-privileged policy is working" {
+    # Launch privileged pod in default namespace (should fail)
+    kubefail_privileged run pod-privileged --image=registry.k8s.io/pause --privileged
+
+    # Launch privileged pod in the ignored namespace (should work according to helm chart options)
+    kubectl create ns shouldbeignored
+    kubectl run pod-privileged --image=registry.k8s.io/pause --privileged --namespace shouldbeignored
```

Comment:
Please use rancher/pause:3.2 image for pause, registry.k8s.io/pause does not have arm version.
See https://github.com/kubewarden/kubewarden-end-to-end-tests/pull/155/commits/4e036defca83411039f1fecebc0f772970fc2d7a for details.


```suggestion
    kubectl run pod-privileged --image=rancher/pause:3.2 --privileged --namespace shouldbeignored
```